setup.py

```diff
@@ -35,31 +34,34 @@
 
 PY3 = sys.version_info[0] == 3
 
-with io.open('README.md', encoding='utf-8') as f:
-    long_description = f.read()
+try:
+    FileNotFoundError
```

Comment:
I think better to show, that it's python version dependent:
```
if not PY3:
    FileNotFoundError = IOError
```

Reply:
Good point.

Reply:
Fixed.